spec/cli/start.spec.js

```diff
@@ -128,4 +133,20 @@ describe('hoodie start', function() {
     });
   });
 
+  describe('$ hoodie start --force', function() {
+    it('should try to start the app with --force', function() {
+      var args = {
+        noBrowser: undefined,
+        www: undefined,
+        sudo: undefined,
+        'custom-ports': undefined,
+        verbose: undefined,
+        force: true
+      };
+      cli.argv({ _: ['start'], force: true });
+      expect(hoodie.start.args[0][0]).to.eql(args);
```

Comment:
we might wanna add a case here where we write a pid file in the test first to simulate a crashed hoodie-server and then make sure we still start.